src/components/views/rooms/EventTile.tsx

```diff
@@ -178,13 +178,6 @@ export function getHandlerTile(ev: MatrixEvent): string {
         }
     }
 
-    if (
-        M_POLL_START.matches(type) &&
```

Comment:
Was this part of the condition important?

Reply:
No, it was just - "if this is a poll event, and we don't have the polls feature, bail out."